chemprop/models/model.py

```diff
@@ -167,17 +166,11 @@ def validation_step(self, batch: TrainingBatch, batch_idx: int = 0):
 
         Z = self.fingerprint(bmg, V_d, X_d)
         preds = self.predictor.train_step(Z)
-        loss = self.metrics[-1](preds, targets, mask, weights, lt_mask, gt_mask)
-        self.log(
-            "val_loss",
-            loss,
-            batch_size=len(batch[0]),
-            prog_bar=True,
-            sync_dist=distributed.is_initialized(),
-        )
+        self.metrics[-1](preds, targets, mask, weights, lt_mask, gt_mask)
```

Comment:
This line does nothing (?)

Reply:
I don't think this affects readability enough to be worth changing

Reply:
Yes it does, which is why torchmetrics recommends doing it this way.